app/code/Magento/InventoryConfigurationApi/Api/GetBackorderStatusConfigurationValueInterface.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryConfigurationApi\Api;
+
+/**
+ * All APIs to retrieve Backorder Config Value on the level of SourceItem/Source/Globally
+ * @api
+ */
+interface GetBackorderStatusConfigurationValueInterface
+{
+    /**
+     * @param string $sku
+     * @param string $sourceCode
+     * @return int|null
+     */
+    public function forSourceItem(string $sku, string $sourceCode): ?int;
+
+    /**
+     * @param string $sourceCode
+     * @return int|null
+     */
+    public function forSource(string $sourceCode): ?int;
+
+    /**
+     * @param string $sourceCode
+     * @return int|null
+     */
+    public function forGlobal(): int;
+
+    /**
+     * Method bound to web API.
+     *
+     * @param string|null $sku
+     * @param string|null $sourceCode
+     * @return int|null
+     */
+    public function execute(string $sku = null, string $sourceCode = null): ?int;
```

Comment:
I need a method to map to a route which is not one of the above Command Facades because to reduce routes we decided to use one method which proxies to Command Facades.
Since the class name already expresses the intention, what would be a better name for you?

Reply:
Should we look for ways to extract this part related to Web API?

Reply:
Yes, an `InventoryConfigurationWebapi` module would be the perfect candidate I guess.

But that doesn't solve the `execute` naming issue, given we still want to go with one route per option, differentiating based on parameters in the body.

What would be an appropriate naming for the function called by the get and set web API?